File: src/main/java/aurocosh/divinefavor/common/network/message/sever/petrification/MessagePetrificationCure.java
package aurocosh.divinefavor.common.network.message.sever.petrification;

import aurocosh.divinefavor.common.network.base.WrappedServerMessage;
import aurocosh.divinefavor.common.potions.common.ModCurses;
import net.minecraft.entity.player.EntityPlayerMP;

public class MessagePetrificationCure extends WrappedServerMessage {
	public MessagePetrificationCure() {}

    @Override
    protected void handleSafe(EntityPlayerMP serverPlayer) {
        serverPlayer.removePotionEffect(ModCurses.petrification);
    }
}

File: src/main/kotlin/aurocosh/divinefavor/common/network/message/sever/petrification/MessagePetrificationCure.kt
package aurocosh.divinefavor.common.network.message.sever.petrification

import aurocosh.autonetworklib.network.base.WrappedServerMessage
import aurocosh.divinefavor.common.potions.common.ModCurses
import net.minecraft.entity.player.EntityPlayerMP

class MessagePetrificationCure : WrappedServerMessage() {
    override fun handleSafe(serverPlayer: EntityPlayerMP) {
        serverPlayer.removePotionEffect(ModCurses.petrification)
    }
}
